### ISSUE TITLE
Concat is ignored when using slash commands

### ISSUE THREAD

**Issue by stijnb1234** [Kaktushose/jda-commands#72]
I have this command:

```java
@Command(value = "reminder", desc = "Set a reminder")
public void onReminder(CommandEvent event, @Param("The time to send the reminder at") String time, @Concat @Param("The message to send with the reminder") String message) {
}
```

And when I use `/reminder 3h Hello there!` the message variable only contains `Hello`.

I'm on v2.3.0-alpha-1.

**Comment by Kaktushose** [comment]
Fixed with 09702a5de4218bff142bcca8b6bf5646e871ff05 Will be part of the next release, planned for tomorrow. 

Also, `@Concat` isn't needed for slash commands

**Comment by stijnb1234** [comment]
> Fixed with [09702a5](https://github.com/Kaktushose/jda-commands/commit/09702a5de4218bff142bcca8b6bf5646e871ff05) Will be part of the next release, planned for tomorrow.
> 
> Also, `@Concat` isn't needed for slash commands

Yes, I assumed that, but it didn't work without it at first either. :P

Thanks for the super quick response and fix.

**Comment by Kaktushose** [comment]
quarantine lifestyle 😅
